.openapidoc/config.js

```diff
@@ -72,7 +72,7 @@ function calculateSpecDetails(specFile) {
 }
 
 function calculateSpecVersion(specFile) {
-  return yaml.safeLoad(fs.readFileSync(specFile, "utf8")).info.version;
+  return yaml.load(fs.readFileSync(specFile, "utf8")).info.version;
```

Comment:
here is the updated method according to https://github.com/nodeca/js-yaml/blob/master/migrate_v3_to_v4.md#safeload-safeloadall-safedump--load-loadall-dump